src/olympia/amo/tests/__init__.py

```diff
@@ -857,7 +857,7 @@ def version_factory(file_kw=None, **kw):
         ApplicationsVersions.objects.get_or_create(application=application,
                                                    version=ver, min=av_min,
                                                    max=av_max)
-    if addon_type != amo.ADDON_PERSONA and file_kw is not False:
+    if addon_type != amo.ADDON_PERSONA and file_kw is not None:
```

Comment:
I believe using `False` here is wrong. I didn't pass `file_kw` yet it was still `True`, but `None`.

Reply:
Hm, but it causes lots of test failures.

Reply:
I believe `False` is right. It's here to give you an option to prevent the creation of a file, if you explicitly pass `file_kw=False` to the function. Which is weird and should probably be removed, but we'd need to fix any code doing that.

In any case, don't change it to `None`.

Reply:
Hm, ok. This is quite confusing.